data/resources.json

```diff
@@ -310,7 +320,7 @@
         {
           "name": "Udemy Beginners Guide to Python",
           "url": "https://blog.udemy.com/python-tutorial/",
-          "description": "This is a guide developed for beginners by Udemy gives a short introduction to Python essentials and core concepts."
+          "description": "This guide developed for beginners by Udemy gives a short introduction to Python essentials and core concepts."
```

Comment:
Since we are following a similar format for all the Udemy resources I think it would be consistent to have commas surrounding the "developed for beginners by Udemy" like you have in the other 3 resources. Other than that :+1: